apps/src/templates/amazonFutureEngineerEligibility/amazonFutureEngineerAccountConfirmation.jsx

```diff
@@ -43,13 +58,13 @@ export default class AmazonFutureEngineerAccountConfirmation extends React.Compo
           if you already have one.
         </div>
         <div style={styles.body}>
-          Already have a Code.org account? <a href={SIGN_IN_URL}>Sign in.</a>
+          Already have a Code.org account?{' '}
+          <a onClick={this.signInButtonPress}>Sign in.</a>
```

Comment:
Removing the href may have changed the style of this link.  I think, in this case, you could avoid that with `href="#"` or something - I don't feel bad about changing the hash since we're about to navigate.

Reply:
Nice, main difference I see is that when you hover it gives you the clicky finger with an href, and the text cursor without it.

Reply:
fwiw, you can use `cursor: pointer` to show the clicky finger again.